direct/core/src/main/java/cz/o2/proxima/direct/core/OnlineAttributeWriter.java

```diff
@@ -67,4 +67,12 @@ default void rollback() {
   @SuppressWarnings("unchecked")
   @Override
   Factory<? extends OnlineAttributeWriter> asFactory();
+
+  default boolean isTransactional() {
+    return false;
+  }
+
+  default TransactionalOnlineAttributeWriter toTransactional() {
+    throw new UnsupportedOperationException();
```

Comment:
shouldn't we create a `TransactionalOnlineAttributeWriter` wrapper here as a default?

Reply:
No, because it has methods to start transaction and that would be a no-op in the case of non-transactional attributes.